src/sidebar/util/test/copy-to-clipboard-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { copyText } = require('../copy-to-clipboard');
+
+describe('copy-to-clipboard', () => {
+  beforeEach(() => {
+    sinon.stub(document, 'execCommand');
+  });
+
+  afterEach(() => {
+    document.execCommand.restore();
+  });
+
+  describe('copyText', () => {
+    /**
+     * Returns the temporary element used to hold text being copied.
+     */
+    function tempSpan() {
+      return document.querySelector('.copy-text');
+    }
+
+    it('copies the passed text to the clipboard', () => {
+      // We can't actually copy to the clipboard due to security restrictions,
+      // but we can verify that `execCommand("copy")` was called and that the
+      // passed text was selected at the time.
+      const testStr = `test string ${Math.random()}`;
```

Comment:
Can you help me understand why this string needs a random element? I have a hunch it has something to do with verifying that this exactly this string (and not one that is similar to it/static) ends up in the resulting selection but I'm not following the thread of how that could be possible.

Reply:
I was originally thinking, "what if something was left over from a previous test?", but it isn't needed. I'll just add something to the test setup to make sure nothing is selected at the start of the test.